### ISSUE TITLE
warframe-market API update

### ISSUE THREAD

**Issue by lyuben-todorov** [tarekis/warframe-rewards-evaluator#3]
Warframe-market public api was apparently changed. I wasn't familiar with the old models but judging by the code this would probably issue a big rewrite.

**Comment by tarekis** [comment]
Yeah, no idea of what changed, but if it did it's probably gonna break, however the screen reading times of the application are utter trash as of now anyways and I can't really make any progress until I get that done anyways, the whole project kind of waits for a new OCR API which some @nexus-devs devs are working on, but I haven't been on it for some while anyways...

**Comment by TobiTenno** [comment]
oh, the warframe.market api is a little bit (not a ton) better. i'm happy to help figure out how to get specific data points if need be
